tests/suites/test_suite_mpi.function

```diff
@@ -188,16 +188,16 @@ void mbedtls_mpi_read_binary( data_t * buf, int radix_A, char * input_A )
 
     TEST_ASSERT( mbedtls_mpi_read_binary( &X, buf->x, buf->len ) == 0 );
     TEST_ASSERT( sign_is_valid( &X ) );
-    TEST_ASSERT( mbedtls_mpi_write_string( &X, radix_A, str, sizeof( str ), &len ) == 0 );
-    TEST_ASSERT( strcmp( (char *) str, input_A ) == 0 );
+    TEST_ASSERT( mbedtls_mpi_write_string( &X, 16, str, sizeof( str ), &len ) == 0 );
+    TEST_ASSERT( strcasecmp( (char *) str, input_A ) == 0 );
```

Comment:
nice :)

Reply:
I was not aware, I think it would be better to use upper case data when necessary to avoid any possible issues with `strcasecmp()`.

Reply:
hmm, did we get a consensus on this?

Reply:
I'll revert this to `strcmp()` to avoid non-standard issues, and to match the expected output of `mbedtls_mpi_write_string()`.